background.js

```diff
@@ -1,36 +1,45 @@
-chrome.browserAction.setBadgeText({text: 'OFF'});
-chrome.browserAction.setBadgeBackgroundColor({color: '#4688F1'});
+chrome.browserAction.setBadgeText({ text: "OFF" });
+chrome.browserAction.setBadgeBackgroundColor({ color: "#4688F1" });
 let reloads_holder = {};
 
 chrome.storage.onChanged.addListener((whatchanged, area) => {
-    if(area==="local"){
-        let keys = Object.keys(whatchanged);
-        keys.forEach(element => {
-            console.log(whatchanged[element]);
-            if(whatchanged[element].newValue){
-                reloads_holder[element] = setInterval(()=>{
-                    chrome.tabs.reload(parseInt(element));
-                }, 10000);
-            }
-            else {
-                clearInterval(reloads_holder[element]);
-            }
-        });
+  console.log(area);
+  if (area === "local") {
+    let keys = Object.keys(whatchanged);
+    keys.forEach((element) => {
+      console.log(element);
+      console.log(whatchanged[element].newValue);
 
-        
-    }
+      if (whatchanged[element].newValue) {
+        reloads_holder[element] = setInterval(() => {
+          chrome.tabs.reload(parseInt(element), () => callback(element));
```

Comment:
Callback is undefined?

Reply:
yes, removed it. My bad i didnt refactor the code.